docs/how-to/dummy-data.md

```diff
@@ -0,0 +1,144 @@
+# How to use dummy data in an ehrQL dataset definition
+
+Because OpenSAFELY doesn't allow direct access to individual patient records, ehrQL allows you
+to work with dummy data for developing analytic code on your own computer.
+
+There are 3 ways to use dummy data in ehrQL.
+
+1. Let ehrQL [generate your dummy dataset from your dataset definition](#generate-a-dummy-dataset-from-a-dataset-definition)
+
+1. Supply a [dummy dataset file](#supply-your-own-dummy-dataset-file)
+
+1. Supply a [set of dummy tables](#supply-your-own-dummy-tables)
```

Comment:
Massive nitpick! Make list items consistent with headings. I'd suggest using the following as list items and headings:

* Let ehrQL generate a dummy dataset from your dataset definition
* Supply your own dummy dataset
* Supply your own dummy tables